packages/browser-data-collector/src/report.ts

```diff
@@ -51,14 +51,26 @@ export class ReportImpl implements Report {
 		}
 	}
 
-	private async start() {
-		await Promise.all( this.startCollectors.map( ( collector ) => collector( this ) ) );
+	private async runCollectors( collectors: Collector[] ) {
+		return Promise.all(
+			collectors.map( ( collector ) => {
+				try {
+					collector( this );
+				} catch {
+					// Swallow the error to make sure a single collector doesn't break the whole report
```

Comment:
Could there be any benefit to logging these errors at least?

Reply:
Done